assets/src/components/App.js

```diff
@@ -1,26 +1,36 @@
-const template = `
-<div>
-  <h1>{{ helloWorld }}</h1>
-</div>`;
-
-import zdClient from '../libs/ZDClient.js';
+import zdClient from './../libs/ZDClient.js';
+import appTemplate from './AppTemplate.js';
```

Comment:
have you tested installing the app? absolute paths didn't work for me once the app is installed